apps/codeforafrica/src/components/NavBar/DesktopNavigation/index.js

```diff
@@ -0,0 +1,50 @@
+import { Grid } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+/* eslint-disable import/no-unresolved */
+import cfaLogo from "@/codeforafrica/assets/CfA logo.svg?url";
+import ImageIcon from "@/codeforafrica/components/ImageIcon";
+import NavBarNavList from "@/codeforafrica/components/NavBarNavList";
+
+const DesktopNavigation = React.forwardRef(function DesktopNavigation(
+  props,
+  ref
+) {
+  const { menu } = props;
+
+  return (
+    <Grid
+      container
+      justifyContent="space-between"
+      alignItems="center"
+      direction="row"
+      ref={ref}
+      sx={{ display: { xs: "none", md: "flex" } }}
+    >
+      <Grid item xs={4}>
+        <ImageIcon src={cfaLogo} alt="Logo" width="136px" height="61px" />
+      </Grid>
+      <Grid item xs={8} container justifyContent="flex-end">
+        <Grid item>
+          <NavBarNavList menu={menu} {...props} />
+        </Grid>
+      </Grid>
```

Comment:
```suggestion
      <Grid item>
        <ImageIcon src={cfaLogo} alt="Logo" width="136px" height="61px" />
      </Grid>
      <Grid item>
        <Box sx={{ justifyContent="flex-end" }}>
          <NavBarNavList menu={menu} direction="row" />
        </Box>
      </Grid>
```

1. I think we can just let everything take it's natural size.
2. Other than `direction`, should there be any other prop passed to `NavBarNavList`?